config/locales/provider_interface/references.yml

```diff
@@ -4,7 +4,7 @@ en:
       card_title:
         school_based: School experience reference from %{name}
         academic: Academic reference from %{name}
-        professional: Academic reference from %{name}
+        professional: Profissional reference from %{name}
```

Comment:
slight typo!

Reply:
20 minutes to spot the bug. 1 second to commit a typo. ✌️ Fixed!